File: audiovideo/src/main/java/com/meitu/audiovideo/MainActivity.java
package com.meitu.audiovideo;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import com.meitu.audiovideo.audio.AudioPlayerActivity;
import com.meitu.audiovideo.video.VideoPlayerActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void startAudio(View view) {
        startActivity(new Intent(this, AudioPlayerActivity.class));
    }

    public void startVideo(View view) {
        startActivity(new Intent(this, VideoPlayerActivity.class));
    }
}

File: learnffmpeg/src/main/java/com/meitu/learnffmpeg/MainActivity.kt
package com.meitu.learnffmpeg

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import com.meitu.learnffmpeg.media.FFMediaPlayer

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val textView = findViewById<TextView>(R.id.textView)
        textView.text = "FFmpeg Version Info:\n" + FFMediaPlayer.getFFmpegVersion()

        val nativePlayer = findViewById<Button>(R.id.nativePlayer)
        nativePlayer.setOnClickListener {
            startActivity(Intent(this, NativeMediaPlayerActivity::class.java))
        }
    }
}
